src/Connection.php

```diff
@@ -420,6 +420,24 @@ public function prepareBindings(array $bindings)
         return $bindings;
     }
 
+    /**
+     * @inheritDoc
+     */
+    protected function escapeBool($value)
+    {
+        return $value ? 'true' : 'false';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function escapeString($value)
+    {
+        return str_contains($value, "\n")
+            ? 'r"""' . addcslashes($value, '"') . '"""'
+            : '"' . addcslashes($value, '"') . '"';
```

Comment:
Is there any documentation that can confirm if this process is correct?
I did not understand why you are doing this....

Reply:
see https://cloud.google.com/spanner/docs/reference/standard-sql/lexical#quoted_literals

Reply:
thanks!